commands/ath.js

```diff
@@ -54,6 +54,11 @@ exports.run = (client, message, args) => {
       }
     }
 
+	if(alchemiter==true && quick == false && client.traitcall.traitCheck(client,charid,"COMPUTER")[1])
```

Comment:
Change this check to only require the computer set check, since we want this to work when there is no alchemiter.